tests/unit/provisioner/test_vagrantprovisioner.py

```diff
@@ -62,8 +67,7 @@ def molecule_file(tmpdir, request):
 
     pbook = d.join(os.extsep.join(('playbook', 'yml')))
     data = [{'hosts': 'all', 'tasks': [{'command': 'echo'}]}]
-
-    pbook.write(data)
+    pbook.write(yaml.dump(data))
```

Comment:
Ah, right, I think it's the anyconfig stuff that introduced the extra info into the molecule dicts when doing a dump.